src/content/reference/fleet-management/migration_to_cluster_api.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Migration to Cluster API
+description: A explanation on how the migration from our old Vintage management clusters to Cluster API works.
+weight: 20
+last_review_date: 2024-03-14
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - What are the requirements for migrating a cluster to Cluster API?
+  - What are the recommendations for a smooth migration?
+---
+
+In Giant Swarm we have used Kubernetes to build platforms since our beginning. We started with a custom solution to manage the lifecycle of the clusters due to the lack of a standard way out of there. It has worked well for us and our customers, but we learnt many companies were resolving the same problem over and over again, so we foster and push for joining forces and build a standard approach. [Cluster API]({{< relref "/reference/fleet-management/introduction" >}}) is backed by the Kubernetes community and covers different providers like AWS, Azure, GCP, and others.
```

Comment:
Could you explain the meaning of `due to lack of a standard way out of there`? I interpret it as `finding a way out of a given situation`, but it's not clear to me what the situation is from the paragraph.

Reply:
Well in 2015 we created our own solution since there was no CAPI. I will make it clearer